### PR TITLE
Output as text

### PR DESCRIPTION
**Pull request by lukaas33**
Added option to output the simplified relations as text instead of as a dot image.

This can be useful for displaying the relations in another visualisation or doing other operations on the data.

### CODE REVIEW

File: README.md
Comment (marksantcroos) [inline README.md:1]: Can you merge master to get in the changes from the other branch?

Comment (lukaas33) [reply README.md:1]: The branch can be merged with master.

**Comment by marksantcroos** [comment]
Why is this one closed now?

**Comment by lukaas33** [comment]
I guess I pressed a wrong button in my editor.
I can reopen it in a new PR.

**Pull request closed without merging**